StaticAnalyzer/views/android/VirusTotal.py

```diff
@@ -79,12 +80,16 @@ def get_result(self, file_path, file_hash):
             if report:
                 if report['response_code'] == 1:
                     print "[INFO] VirusTotal: " + report['verbose_msg']
-                    return report
-            print "[INFO] VirusTotal: " + report['verbose_msg']
-            print "[INFO] VirusTotal: file upload"
-            upload_response = self.upload_file(file_path)
-            if upload_response:
-                print "[INFO] VirusTotal: " + upload_response['verbose_msg']
-            return upload_response
+                    return report                  
+            if settings.VT_UPLOAD:
+                print "[INFO] VirusTotal: file upload"
+                upload_response = self.upload_file(file_path)
+                if upload_response:
+                    print "[INFO] VirusTotal: " + upload_response['verbose_msg']
+                return upload_response
+            else:
+                print "[INFO] MobSF: Upload not done VT_UPLOAD set to False in settings.py"
+                report = {"verbose_msg": "No Report file upload disabled"}
```

Comment:
Reword to `Scan Not performed, VirusTotal file upload disabled in settings.py`

Reply:
done